packages/playground/src/components/profile_manager/ConnectWallet.vue

```diff
@@ -124,16 +125,18 @@
           label="Email"
           placeholder="email@example.com"
           :loading="loadEmail"
-          :disabled="creatingAccount || connecting || loadEmail || mnemonicInput?.status !== ValidatorStatus.Valid"
+          :disabled="isEmailDisabled"
           autocomplete="off"
+          @blur="() => emailInput?.validate()"
+          @update:model-value="() => emailInput?.validate()"
         />
       </input-validator>
       <!-- Passwords -->
       <WalletPassword
         id="password-input"
         v-model="password"
         mode="Create"
-        :disabled="creatingAccount || connecting"
+        :disabled="creatingAccount || connecting || !isMnemonicValid"
```

Comment:
Why do we disable those if the mnemonic is empty? 
The idea behind disabling the email field is to avoid overwriting the entered email after getting the stored email. This case is on password fields, so I don't think we should disable them

Reply:
if we don't have a valid mnemonic, why do we need it enabled? It's useless